core/arch/atomic_exports.h

```diff
@@ -625,34 +636,67 @@ atomic_dec_becomes_zero(volatile int *var)
  * so we require alignment here.
  */
 /* XXX i#1551: should we allow infinite loops for those ATOMIC ops */
-#        define ATOMIC_INC_suffix(suffix, var)                                     \
-            __asm__ __volatile__("   dmb ish                        \n\t"          \
-                                 "1: ldrex" suffix " r2, %0         \n\t"          \
-                                 "   add" suffix " r2, r2, #1     \n\t"            \
-                                 "   strex" suffix " r3, r2, %0     \n\t"          \
-                                 "   cmp   r3, #0                   \n\t"          \
-                                 "   bne   1b                       \n\t"          \
-                                 "   cmp   r2, #0" /* for possible SET_FLAG use */ \
-                                 : "=Q"(var)       /* no offset for ARM mode */    \
-                                 :                                                 \
+#        define ATOMIC_INC_suffix(suffix, var)                            \
+            __asm__ __volatile__("   dmb ish                        \n\t" \
+                                 "1: ldrex" suffix " r2, %0         \n\t" \
```

Comment:
style consistency nit: this uses `suffix` instead of  `#suffix` and that arg is a string. Just different than the new ones added here.

Reply:
Searching the file: there are 20-odd that use a string and basically none without a string until the ones I added, so I changed the new non-string to instead use a string, for consistency.